test/index.test.js

```diff
@@ -10,7 +10,7 @@ test('Should shortcircuits preflight requests', t => {
   const fastify = Fastify()
   fastify.register(cors)
 
-  fastify.options('/', (req, reply) => {
+  fastify.get('/', (req, reply) => {
```

Comment:
why this change?

Reply:
Leaving this as `options`, would override the route added inside fastify-cors and that is not the intention of the test.

I changed these to `get` as just kind of a hacky fix to the tests, but perhaps it would be better just to remove these routes... That more sense, if the route is not "short-circuited" it would finish with a 404. I'll do that.